src/components/menu-component.jsx

```diff
@@ -68,16 +70,34 @@ function menuComponents(items, parentComponent) {
 class MainMenu extends React.Component {
   constructor(props) {
     super(props)
+    this.state = {
+      isModalOpen: false,
```

Comment:
`isDeleteNotebookDialogOpen` is probably a better naming convention than this, for now.

We will likely refactor this down the road to make room for other dialogs, but for until we refactor how we're dealing with menus I think this approach works just fine.